nautobot/dcim/forms.py

```diff
@@ -2257,6 +2257,11 @@ class DeviceFilterForm(
         label="Has a primary IP",
         widget=StaticSelect2(choices=BOOLEAN_WITH_BLANK_CHOICES),
     )
+    has_interfaces = forms.NullBooleanField(
+        required=False,
+        label="Has interfaces",
+        widget=StaticSelect2(choices=BOOLEAN_WITH_BLANK_CHOICES),
+    )
     virtual_chassis_member = forms.NullBooleanField(
         required=False,
         label="Virtual chassis member",
```

Comment:
Don't the `console_ports`, `console_server_ports`, etc. NullBooleanFields need to be renamed to `has_*` to continue working?